.github/workflows/fix-linting.yml

```diff
@@ -13,13 +13,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{ github.event.pull_request.head.sha }}
-
-      # Action runs on the issue comment, so we don't get the PR by default
-      - name: Checkout Pull Request
-        run: hub pr checkout ${{ github.event.issue.number }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
This doesn't work, as the action runs on the issue comment being added and not the PR event. So there is no `github.event.pull_request`.